webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,17 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand",function(){})
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.style.overflow = 'hidden';
+        this.style.height = 0;
```

Comment:
Do you mean we need to adjust the height if the user delete some stuff from their textarea comment? `scrollHeight` will account for that. Have you tested without setting height to 0?